plugin/storage/es/spanstore/dbmodel/from_domain_test.go

```diff
@@ -108,6 +108,32 @@ func TestTagMap(t *testing.T) {
 	assert.Equal(t, tagsMap, dbSpan.Process.Tag)
 }
 
+func TestNilProcess(t *testing.T) {
+	tags := []model.KeyValue{
+		model.String("foo", "foo"),
+		model.Bool("a", true),
+		model.Int64("b.b", 1),
+	}
+
+	spanWithNilProcessTags := model.Span{Tags: tags, Process: &model.Process{Tags: nil}}
+	spanWithNilProcess := model.Span{Tags: tags, Process: nil}
+
+	converter := NewFromDomain(false, nil, ":")
+	dbSpanWithNilTags := converter.FromDomainEmbedProcess(&spanWithNilProcessTags)
+	dbSpanWithNilProcess := converter.FromDomainEmbedProcess(&spanWithNilProcess)
+
+	assert.Equal(t, 3, len(dbSpanWithNilTags.Tags))
+	assert.Equal(t, 3, len(dbSpanWithNilProcess.Tags))
+	assert.Equal(t, 0, len(dbSpanWithNilTags.Process.Tags))
+	assert.Equal(t, 0, len(dbSpanWithNilProcess.Process.Tags))
+
+	tagsMap := map[string]interface{}(nil)
```

Comment:
not sure if this was expected but this is what I got from the `convertKeyValuesString` func if the Tags was nil.